todoman/formatters.py

```diff
@@ -180,7 +185,7 @@ def _parse_datetime_naive(self, dt):
             pass
 
         try:
-            return datetime.datetime.strptime(dt, self.date_format)
+            return datetime.datetime.strptime(dt, self.date_format).date()
```

Comment:
Can you add a docstring to this method? The return value is not as obvious as the name implies any more.